### DISCUSSION TITLE
How do I test a new script

### DISCUSSION THREAD

**Discussion by yry-dev** [community-scripts/ProxmoxVE#197] [Q&A] [answered]
I am attempting to write a new install script for Komga so I can contribute it to the project.  I have forked the repository and I think I have an install script for a new LXC but how do you test this locally?

Here is what I have:
https://github.com/DysfunctionalProgramming/ProxmoxVE/blob/main/ct/komga.sh
https://github.com/DysfunctionalProgramming/ProxmoxVE/blob/main/install/komga-install.sh

I tried doing this but it seemingly created the lxc but I couldn't login to it and it didnt install the service.
```
bash -c "$(wget -qLO - https://raw.githubusercontent.com/DysfunctionalProgramming/ProxmoxVE/refs/heads/main/ct/komga.sh)"
```

Are there instructions or a guide for how to develop new scripts / test new scripts etc?

**Comment by MickLesk** [comment, marked as answer]
You need to increase the build.func & install.func (Switch the links to your local Repo).

Take a Look at my Proxmox_DEV Repo in misc (its an old old clone from tteck) - Check the latest Updates (build.func / install.func)
